src/screens/DiscoveryScreenV2/elements/DiscoveryTab.js

```diff
@@ -47,38 +67,41 @@ const styles = StyleSheet.create({
     height: 48,
     backgroundColor: colors.white
   },
-  tabItem: (index) => ({
-    flex: 1,
-    width: width / 4,
+  tabItem: (numTabs) => ({
+    width: numTabs === 4 ? undefined : windowWidth / numTabs,
     justifyContent: 'center',
+    alignItems: 'center',
     height: '100%',
-    paddingLeft: index !== 1 ? 20 : 0
+    paddingHorizontal: numTabs === 4 ? 20 : undefined
   }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `tabItem` style now depends on the number of tabs. This could lead to unexpected behavior if the number of tabs is not 3 or 4. Consider adding a default case to handle other numbers of tabs.
```diff
- tabItem: (index) => ({
-   flex: 1,
-   width: width / 4,
-   justifyContent: 'center',
-   height: '100%',
-   paddingLeft: index !== 1 ? 20 : 0
- }),
+ tabItem: (numTabs) => ({
+   width: numTabs === 4 ? undefined : windowWidth / numTabs,
+   justifyContent: 'center',
+   alignItems: 'center',
+   height: '100%',
+   paddingHorizontal: numTabs === 4 ? 20 : undefined
+ }),
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->